docs/source/index.rst

```diff
@@ -96,7 +96,7 @@ Composer is part of the broader Machine Learning community, and we welcome any c
    :caption: Methods Library
 
    method_cards/methods_overview.rst
-   method_cards/agc.md
+   method_cards/gradient_clipping.md
```

Comment:
Can you move it down to the correct position in alphabetical order?

Reply:
done here 
47c25c1